.github/workflows/upgrade.yaml

```diff
@@ -0,0 +1,57 @@
+name: Update vCluster dep
```

Comment:
Let's add run details, since this is dispatch triggered

`run-name: ${{ github.actor }} is upgrading vcluster dependencies`

Reply:
added